core/controller/src/main/scala/whisk/core/controller/Triggers.scala

```diff
@@ -146,9 +147,14 @@ trait WhiskTriggersApi extends WhiskCollectionAPI {
                 case Failure(t) =>
                   logging.error(this, s"[POST] storing trigger activation $triggerActivationId failed: ${t.getMessage}")
               }
+            complete(Accepted, triggerActivationId.toJsObject)
+          } else {
+            logging
+              .info(
+                this,
+                s"[POST] trigger without an active rule was activated; no trigger activation record created for $triggerActivationId")
+            complete(NoContent)
```

Comment:
we could add a message here so that the HTTP resonse status code is NoContent and the message is informative: Trigger has no associated rules that are active.